### PR TITLE
Publish 3.3.11 release notes

### PR DESCRIPTION
**Pull request by alexanderahn**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208216813765154

### CODE REVIEW

File: docs/release_notes/2024/sept2024.md [hunk 1]
```diff
@@ -9,6 +9,27 @@ tags:
   - september
 ---
 
+## 3.3.11.\* (September 20, 2024)
+
+- Prophecy Python libs version: 1.9.14
+- Prophecy Scala libs version: 8.1.4
+
+### Features {#Features3311}
```
Comment (kshitiz-suman) [inline docs/release_notes/2024/sept2024.md:17]: @alexanderahn Jinja Syntax feature isn't there ???

Comment (alexanderahn) [reply docs/release_notes/2024/sept2024.md:17]: Does the item on line 21 not refer to it?

File: docs/release_notes/2024/sept2024.md [hunk 2]
```diff
@@ -9,6 +9,27 @@ tags:
...
+
+#### Spark Copilot Enhancements {#Spark3311}
+
+- **Config syntax**: We've unified the config syntax across all project types by supporting Jinja config syntax. This means that you can use `{{foo}}` for variable substitution.
```
Comment (atbida) [inline docs/release_notes/2024/sept2024.md:21]: For[ visual language ](https://docs.prophecy.io/Spark/configuration/#sql-visual-language) we'll need to update the docs.

File: docs/release_notes/2024/sept2024.md [hunk 3]
```diff
@@ -9,6 +9,27 @@ tags:
...
+
+#### SQL Copilot Enhancements {#SQL3311}
+
+- **Lineage run**: We now support SQL run history and grouping for SQL runtime logs. We also allow you to diagnose seed error performance.
+
+- **Parametrized schema**: In some scenarios, you may need to parameterize your location paths using external variables. You can do this by enabling the Schema field parameterization on the Location tab of your Target Model.
+
+### Minor Improvements {#MinorImprovements3311}
+
+- **Deployment APIs**: You can use our external APIs for deployment and monitoring of your Prophecy project.
```
Comment (atbida) [inline docs/release_notes/2024/sept2024.md:31]: Link to API docs when we have them

Comment (alexanderahn) [reply docs/release_notes/2024/sept2024.md:31]: @pankajb23 is there a link to the APIs that we can share?

File: docs/release_notes/2024/sept2024.md [hunk 4]
```diff
@@ -9,6 +9,27 @@ tags:
...
+
+### Features {#Features3311}
+
+#### Spark Copilot Enhancements {#Spark3311}
+
+- **Config syntax**: We've unified the config syntax across all project types by supporting Jinja config syntax. This means that you can use `{{foo}}` for variable substitution.
+
+#### SQL Copilot Enhancements {#SQL3311}
+
+- **Lineage run**: We now support SQL run history and grouping for SQL runtime logs. We also allow you to diagnose seed error performance.
```
Comment (alexanderahn) [inline docs/release_notes/2024/sept2024.md:25]: To create docs from 9/4 demo recording 30min~, by Chirag.

Comment (alexanderahn) [reply docs/release_notes/2024/sept2024.md:25]: https://app.asana.com/0/1203457878958265/1208338200470217

File: docs/release_notes/2024/sept2024.md (hunk 2 above)
Comment (alexanderahn) [reply docs/release_notes/2024/sept2024.md:21]: Currently working on. Thank you for the comment and link.

Comment (alexanderahn) [reply docs/release_notes/2024/sept2024.md]: Added more details.

File: docs/release_notes/2024/sept2024.md (hunk 4 above)
Comment (alexanderahn) [reply docs/release_notes/2024/sept2024.md:31]: Added more details.

File: docs/release_notes/2024/sept2024.md (hunk 3 above)
Comment (alexanderahn) [reply docs/release_notes/2024/sept2024.md:49]: Waiting on respone from Pankaj and rest of the team here and on https://app.asana.com/0/1206327447905631/1208168278656704/f. Also on Slack.

**Comment by alexanderahn** [comment]
Will wait until Monday for reviews on this, https://github.com/SimpleDataLabsInc/prophecy-docs/pull/390, and https://github.com/SimpleDataLabsInc/prophecy-docs/pull/396 to merge.

File: docs/release_notes/2024/sept2024.md (hunk 3 above)
Comment (alexanderahn) [reply docs/release_notes/2024/sept2024.md:49]: Did not receive any response.

**Pull request merged**